packages/wonder-blocks-clickable/src/components/__docs__/clickable.stories.js

```diff
@@ -0,0 +1,296 @@
+                >
+                    <Body>This text is clickable!</Body>
+                </View>
+            )}
+        </Clickable>
+    </View>
+);
+
+Light.parameters = {
+    chromatic: {
+        // Not needed because the default state doesn't test the disabled
+        // clickable behavior.
+        disableSnapshot: true,
+    },
+    docs: {
+        description: {
+            story: "Clickable has a `light` prop which changes the default focus ring color to fit a dark background.",
+        },
+    },
+    backgrounds: {
+        default: "darkBlue",
+    },
+};
+
+/**
+ * Disabled state
+ */
+export const Disabled: StoryComponentType = (args) => (
+    <>
+        <Clickable onClick={() => {}} {...args}>
+            {({hovered, focused, pressed}) => (
+                <View
+                    style={[
+                        styles.clickable,
+                        hovered && styles.hovered,
+                        pressed && styles.pressed,
+                    ]}
+                >
+                    <Body>
+                        Disabled clickable using the default disabled style
+                    </Body>
+                </View>
+            )}
+        </Clickable>
+        <Clickable onClick={() => {}} {...args}>
+            {({hovered, focused, pressed}) => (
+                <View
+                    style={[
+                        styles.clickable,
+                        hovered && styles.hovered,
+                        pressed && styles.pressed,
+                        args.disabled && styles.disabled,
+                    ]}
+                >
+                    <Body>
+                        Disabled clickable passing custom disabled styles
+                    </Body>
+                </View>
+            )}
+        </Clickable>
+    </>
+);
+
+Disabled.args = {
+    disabled: true,
+};
+
+Disabled.parameters = {
+    docs: {
+        description: {
+            story: "Clickable has a `disabled` prop which prevents the element from being operable. Note that the default disabled style is applied to the element, but you can also pass custom styles to the children element by passing any `disabled` styles (see the second clickable element in the example below).",
+        },
+    },
+};
+
+export const ClientSideNavigation: StoryComponentType = () => (
+    <MemoryRouter>
+        <View>
+            <View style={styles.row}>
+                <Clickable
+                    href="/foo"
+                    style={styles.heading}
+                    onClick={() => {
+                        // eslint-disable-next-line no-console
+                        console.log("I'm still on the same page!");
+                    }}
+                >
+                    {(eventState) => (
+                        <LabelLarge>Uses Client-side Nav</LabelLarge>
+                    )}
+                </Clickable>
+                <Clickable href="/foo" style={styles.heading} skipClientNav>
```

Comment:
Is the "Avoids Client-side Nav" example intended to 404 on click? If so, could you add that to the description?

Reply:
Good question! I think I'm going to add an existing link instead.